operators/hack/deployer/vault.go

```diff
@@ -29,7 +32,7 @@ func ReadVaultIntoFile(fileName, address, roleId, secretId, name string) error {
 			"Address": address,
 			"Name":    name,
 		}).
-		WithVariable(vaultTokenName, vaultToken).
+		WithVariable(vaultTokenName, strings.Trim(vaultToken, "\n")).
```

Comment:
Do you have any idea where this `\n` is coming?

Reply:
I think read will return a list, hence it might be ending with newline - this is pre-emptive though.